site/src/pages/docs/advanced-spring-boot-integration.mdx

```diff
@@ -0,0 +1,382 @@
+# Using Spring Boot with Armeria
```

Comment:
How about aligning with other titles?
```suggestion
# Using Armeria with Spring Boot
```

Reply:
<img width="277" alt="스크린샷 2023-07-03 오후 9 08 01" src="https://github.com/line/armeria/assets/24784661/fe766a28-69a0-433e-a53f-6daab8a18ed7">

Thanks! The order of the words in the title is intentional. If the word "Armeria" goes first, the overflowed sentence will be omitted. I think the world "Armeria" is less important because we already knows this document is about Armeria. Therefore, I locate the "Spring Boot" first. Also, I'm thinking about changing the other titles as well. What do you think?

Reply:
Good idea!

Reply:
How about just `Spring Boot integration` then, like other integration pages?

Reply:
Thanks! I think it's good idea!

Reply:
That sounds awesome. 😉